### PR TITLE
Fox be gone

### PR DESCRIPTION
**Pull request by Moribun**
removes kitsune and there dependencies. this was frustrating. I wouldnt be opposed to a double check on it since I had to surgically remove ALOT of stuff. but it ran aight for me with no runtimes.

also as a furry removing a furry I feel there is a pun to be made here but kitsunes suck so *shrug*

also fixed 2 errors related to dwarf beard and diona. because why the fuck not I was already there.

### CODE REVIEW

File: code/modules/language/language_holder.dm [hunk 1]
```diff
@@ -330,8 +330,6 @@ Key procs
 								/datum/language/moffic = list(LANGUAGE_ATOM),
 								/datum/language/calcic = list(LANGUAGE_ATOM),
 								/datum/language/voltaic = list(LANGUAGE_ATOM),
-								/datum/language/nekomimetic = list(LANGUAGE_ATOM),
```
Comment (CPhantasm) [inline code/modules/language/language_holder.dm:333]: I believe there should be a honk comment if you're removing nekomimetic since it's from upstream

File: code/modules/admin/create_mob.dm [hunk 2]
```diff
@@ -29,6 +29,21 @@
 	human.dna.blood_type = random_blood_type()
 	human.dna.features["mcolor"] = "#[random_color()]"
 	human.dna.species.randomize_active_underwear(human)
+	human.dna.features["ethcolor"] = GLOB.color_list_ethereal[pick(GLOB.color_list_ethereal)]
```
Comment (CPhantasm) [inline code/modules/admin/create_mob.dm:32]: Probably also want a honk comment here as well

File: code/modules/language/language_holder.dm (hunk 1 above)
Comment (Inari-Whitebear) [reply code/modules/language/language_holder.dm:333]: but nekomimetic is cat language? we have felinids

Comment (Moribun) [reply code/modules/language/language_holder.dm:333]: ah so neko-jibberish did get pulled out too during the revert too. easy enough to fix back

**Comment by Moribun** [comment]
added back neko speak. it got pulled out with kitsune-whatever-the-fuck language was removed.

File: code/modules/admin/create_mob.dm [hunk 3]
```diff
@@ -29,6 +29,21 @@
 	human.dna.blood_type = random_blood_type()
 	human.dna.features["mcolor"] = "#[random_color()]"
 	human.dna.species.randomize_active_underwear(human)
+	human.dna.features["ethcolor"] = GLOB.color_list_ethereal[pick(GLOB.color_list_ethereal)] //honk
```
Comment (cakexensen) [inline code/modules/admin/create_mob.dm:32]: why was all of this added?

File: code/modules/mob/living/carbon/human/species_types/flypeople.dm [hunk 4]
```diff
@@ -5,7 +5,6 @@
 	species_traits = list(HAS_FLESH, HAS_BONE, TRAIT_ANTENNAE)
 	inherent_traits = list(
 		TRAIT_CAN_USE_FLIGHT_POTION,
-		TRAIT_TACKLING_FRAIL_ATTACKER,
```
Comment (cakexensen) [inline code/modules/mob/living/carbon/human/species_types/flypeople.dm:8]: why is this removed?

File: code/modules/research/techweb/all_nodes.dm [hunk 5]
```diff
@@ -431,10 +431,12 @@
 		"limbdesign_ethereal",
 		"limbdesign_felinid",
 		"limbdesign_lizard",
-		"limbdesign_plasmaman",//honk start - Adding the Skaven and kitsune Designs to the Research
+		"limbdesign_plasmaman",
 		"limbdesign_skaven",
-		"limbdesign_kitsune",//honk end
-	)
+		"limbdesign_kitsune",
+		"limbdesign_plasmaman",
```
Comment (cakexensen) [inline code/modules/research/techweb/all_nodes.dm:437]: remove duplicated lines, make sure honk comment stays in. since our modification is only the skaven line can do a single comment at the end of the line

File: code/modules/surgery/organs/tongue.dm [hunk 6]
```diff
@@ -189,6 +189,46 @@
 				qdel(dropped)
 	qdel(owner)
 
+/obj/item/organ/internal/tongue/fly
```
Comment (cakexensen) [inline code/modules/surgery/organs/tongue.dm:192]: this should not have been readded here

File: code/modules/unit_tests/screenshot_humanoids.dm [hunk 7]
```diff
@@ -17,7 +17,7 @@
 	var/mob/living/carbon/human/dwarf = allocate(/mob/living/carbon/human/dummy/consistent)
 	dwarf.set_species(/datum/species/dwarf)
 	dwarf.facial_hairstyle = "Beard (Dwarf)"
-	dwarf.update_hair(is_creating = TRUE)
+//	dwarf.update_hair(is_creating = TRUE) //don't know what you was doing with this. striked for now. bad arguement.
```
Comment (cakexensen) [inline code/modules/unit_tests/screenshot_humanoids.dm:20]: if there's something wrong with this dorf code make it part of a separate PR, sorry

File: russstation/code/modules/clothing/spacesuits/_spacesuits.dm [hunk 8]
```diff
@@ -1,10 +1,10 @@
 //prevents dionae from wearing space suits
-/obj/item/clothing/head/helmet/space/mob_can_equip(mob/living/M, mob/living/equipper, slot, disable_warning = FALSE, bypass_equip_delay_self = FALSE)
+/obj/item/clothing/head/helmet/space/mob_can_equip(mob/living/M, mob/living/equipper, slot, disable_warning = FALSE, bypass_equip_delay_self = FALSE, ignore_equipped = FALSE)
```
Comment (cakexensen) [inline russstation/code/modules/clothing/spacesuits/_spacesuits.dm:2]: why was this changed? should be a separate PR

File: russstation/code/modules/surgery/organs/external/tails.dm [hunk 9]
```diff
@@ -4,11 +4,4 @@
...
 	wag_flags = WAG_ABLE
-
-/obj/item/organ/external/tail/kitsune
-	name = "Fox tail"
-	desc = "A severed fox tail. are you sure this isn't cursed..?"
-	preference = "feature_kitsune_tail"
-	feature_key = "tail_kitsune"
-	color_source = ORGAN_COLOR_HAIR
-	wag_flags = WAG_ABLE
+	// icon = ""
```
Comment (cakexensen) [inline russstation/code/modules/surgery/organs/external/tails.dm:7]: remove commented line?

**Review by cakexensen — CHANGES_REQUESTED** [review]
not sure what's up with some of these changes, i'm guessing you did something like a `git revert` and it clobbered some stuff unrelated to kitsune? i saw what appeared to be small fixes for dorf and dionae, please do those in a separate PR. i know it's a hassle but since this is a code removal PR we don't want to mix in anything else.
